tests/PySys/smoketest_json_400_samples_10ms/run.py

```diff
@@ -0,0 +1,74 @@
+import os
+import pysys
+from pysys.basetest import BaseTest
+
+import sys
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+
+import time
+
+"""
+Roundtrip test C8y 400 samples 10ms delay
+
+Given a configured system with configured certificate
+When we derive from EnvironmentC8y
+When we run the smoketest for JSON publishing with defaults a size of 400, 10ms delay
+Then we validate the data from C8y
+"""
+
+
+class PySysTest(EnvironmentC8y):
```

Comment:
This test is too similar to the plain `smoketetst_json` test. I'd say that the variation in just a few args can be captured differently with some inheritance, instead of duplicating the same test here.

Reply:
Hi @albinsuresh I had a look at the modes feature and I'm not sure if it is an easy way to run paramerised tests. I also made a( not a yet pushed) proposal that uses inheritance. I'm not sure if it helps. let have a chat about that.